### PR TITLE
Add page number to page nav

### PR DESCRIPTION
**Pull request by LenaESoftwire**
https://trello.com/c/ycKcKhdO/77-add-page-number-to-page-nav-display

### CODE REVIEW

File: WhaleSpotting/react-app/src/components/PageNav.tsx [hunk 1]
```diff
@@ -12,20 +12,22 @@ export default function PageNav({ page, nextPage, previousPage }: pageNavProps):
...
-                text="Next Page"
-                onClick={() => nextPage()}
-                minWidth25={true}
-            />
+            <div className="previous-button" data-testid="previous-page">
+                <Button 
+                    style={Style.secondary}
+                    text="Previous Page"
+                    onClick={() => previousPage()}
+                    hidden={page <= 1 ? true : false}
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/components/PageNav.tsx]:
```suggestion
                    hidden={page <= 1}
```

File: WhaleSpotting/react-app/src/styles/PageNav.scss [hunk 2]
```diff
@@ -1,6 +1,33 @@
 @import "./Constants.scss";
 
 .page-nav {
-    display: flex;
+    display: inline-flex;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/PageNav.scss]: why the change to inline flex?

**Comment by LenaESoftwire** [comment]
PC:
image
image


mobile:
image
image

File: WhaleSpotting/react-app/src/styles/PageNav.scss [hunk 3]
```diff
@@ -1,6 +1,33 @@
...
     justify-content: space-between;
+    width: 100%;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/PageNav.scss]: not needed

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/styles/PageNav.scss (hunk 3 above)
Comment (LenaESoftwire) [reply WhaleSpotting/react-app/src/styles/PageNav.scss]: image
It does affect when I change this

File: WhaleSpotting/react-app/src/styles/PageNav.scss (hunk 2 above)
Comment (LenaESoftwire) [reply WhaleSpotting/react-app/src/styles/PageNav.scss]: fixed

File: WhaleSpotting/react-app/src/styles/PageNav.scss (hunk 3 above)
Comment (oskarwilliams) [reply WhaleSpotting/react-app/src/styles/PageNav.scss]: Fair enough! I am wrong

Comment (LenaESoftwire) [reply WhaleSpotting/react-app/src/styles/PageNav.scss]: Fixed. It was coming with inline-flex

**Review by oskarwilliams — APPROVED** [review]
